include/linux/soundwire/sdw.h

```diff
@@ -363,6 +365,7 @@ struct sdw_dpn_prop {
  * @p15_behave: Slave behavior when the Master attempts a read to the Port15
  * alias
  * @lane_control_support: Slave supports lane control
+ * @lane_maps: Lane mapping for the slave
```

Comment:
presumably only valid if lane_control_support is true?

Reply:
> presumably only valid if lane_control_support is true?

Yes, and changed the description to "Lane mapping for the slave, only valid if lane_control_support is set"